build/build.php

```diff
@@ -394,6 +394,7 @@ function clean_composer(string $dir)
 	'plugins/sampledata/testing/testing.xml',
 	'plugins/sampledata/testing/language/en-GB/en-GB.plg_sampledata_testing.ini',
 	'plugins/sampledata/testing/language/en-GB/en-GB.plg_sampledata_testing.sys.ini',
+    'ruleset.xml',
```

Comment:
```suggestion
	'ruleset.xml',
```
As the build.php is not psr12 yet we should use a tab here like in the lines above and below, and not 4 spaces.

Reply:
You cant do that. .editorconfig contains the rules that ide should follow. It will be a nightmare if we dont follow it.

Reply:
The build folder is configured to be skipped in the ruleset.xml!!! The file is still with tabs. Check the diff on GH and you will see.

Reply:
I am changing it but this is an issue that needs to be resolved by the maintainers

Reply:
Yes, I agree. I have already started to discuss that we should convert the code style of the PHP files in the build folder, even if we not include it in the automated checks for some reason.

Reply:
For the code style thing see #38167 .